### PR TITLE
add pipeline configurations

### PR DESCRIPTION
**Pull request by IsaackMwenda**
This adds the pipeline configuration files. This uses the old structure

### CODE REVIEW

File: configuration/pipeline_config.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    {"RapidProKey": "Recently_Displaced (Time) - imaqal_demog", "PipelineKey": "recently_displaced_time"},
+    {"RapidProKey": "Household_Language (Value) - imaqal_demog", "PipelineKey": "household_language_raw"},
+    {"RapidProKey": "Household_Language (Time) - imaqal_demog", "PipelineKey": "household_language_time"},
+    {"RapidProKey": "District (Value) - imaqal_demog", "PipelineKey": "location_raw"},
+    {"RapidProKey": "District (Time) - imaqal_demog", "PipelineKey": "location_time"}
+  ],
+  "ProjectStartDate": "2020-03-01T00:00:00+03:00",
+  "ProjectEndDate": "2100-01-01T00:00:00+03:00",
+  "FilterTestMessages": true,
+  "MoveWSMessages": false,
```
Comment (as2388) [inline configuration/pipeline_config.json]: We had this switched on immediately in COVID19 (Kenya)

File: src/lib/pipeline_configuration.py [hunk 2]
```diff
@@ -0,0 +1,620 @@
+import json
```
Comment (as2388) [inline src/lib/pipeline_configuration.py:1]: (sorry this wasn't as copy/paste I'd hoped - I had moved this into configuration/ but didn't want to merge until I could test it on a new project. Next time...)

**Review by as2388 — APPROVED** [review]

File: src/lib/pipeline_configuration.py (hunk 2 above)
Comment (IsaackMwenda) [reply src/lib/pipeline_configuration.py:1]: Thanks will move well re arranging the scripts

File: configuration/pipeline_config.json (hunk 1 above)
Comment (IsaackMwenda) [reply configuration/pipeline_config.json]: Thanks have switched it ws and debug mode on to test.

**Pull request merged**